src/codecs/tga/encoder.rs

```diff
@@ -34,12 +37,60 @@ impl error::Error for EncoderError {}
 /// TGA encoder.
 pub struct TgaEncoder<W: Write> {
     writer: W,
+
+    /// Run-length encoding
+    use_rel: bool,
```

Comment:
Why "rel" rather than "rle"?

Reply:
I don't know how I typed that 20 times without noticing.